src/components/newUIComponents/nav/Navbar2.js

```diff
@@ -0,0 +1,53 @@
+import { NavLink } from "react-router-dom";
+import { LuSun } from "react-icons/lu";
+import logo from "../../../assets/logo.svg";
+import React from "react";
+
+export default function Navbar2({ props }) {
+  return (
+    <div className="Navbar2">
+      <div className="brand">
+        <img src={logo} className="brand-img" alt="brand" />
+      </div>
+      <nav className="links">
+        <NavLink
+          to="/"
+          className={({ isActive }) => (isActive ? "link-active" : "link")}
+        >
+          Dashboard
+        </NavLink>
+        <LuSun className="sun" />
+        <NavLink
+          to="/sites"
+          className={({ isActive }) => (isActive ? "link-active" : "link")}
+        >
+          Sites
+        </NavLink>
+        <LuSun className="sun" />
+        <NavLink
+          to="/reports"
+          className={({ isActive }) => (isActive ? "link-active" : "link")}
+        >
+          Reports
+        </NavLink>
+        <LuSun className="sun" />
+        <NavLink
+          to="/alerts"
+          className={({ isActive }) => (isActive ? "link-active" : "link")}
+        >
+          Alerts
+        </NavLink>
+        <LuSun className="sun" />
+        <NavLink
+          to="/manage"
+          className={({ isActive }) => (isActive ? "link-active" : "link")}
+        >
+          Manage
+        </NavLink>
+      </nav>
+      <div className="profile">
```

Comment:
```<Avatar
          name={user.attributes.name}
          size={40}
          textSizeRatio={2}
          round={true}
        />``` provides this

Reply:
the avatar component had some weird positional/box sizing issues I was running into, decide to make my own instead of altering that one and potentially affecting the old UI